R/snapshot.R

```diff
@@ -3,39 +3,33 @@
 # can be scoped to FALSE when hashing is not necessary
 `_renv_snapshot_hash` <- TRUE
 
-#' Snapshot a Project
+#' Record current state of the project library in the lockfile
 #'
-#' Call `snapshot()` to create a **lockfile** capturing the state of a project's
-#' \R package dependencies. The lockfile can be used to later restore these
-#' project's dependencies as required.
+#' @description
+#' Call `renv::snapshot()` to update a [lockfile][lockfiles] with the current
+#' state of dependences in the project library. The lockfile can be used to
```

Comment:
```suggestion
#' state of dependencies in the project library. The lockfile can be used to
```

Still though, I think the documentation should emphasize that `snapshot()` is to "save" as `restore()` is to "load"; that is, `snapshot()` saves the current project state / state of your library paths, and `restore()` restores that state.